cmd/users/list_entity.go

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2023-present Coinbase Global, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package users
+
+import (
+	"fmt"
+
+	"github.com/coinbase-samples/prime-cli/utils"
+	"github.com/coinbase-samples/prime-sdk-go/model"
+	"github.com/coinbase-samples/prime-sdk-go/users"
+
+	"github.com/spf13/cobra"
+)
+
+var listEntityUsersCmd = &cobra.Command{
+	Use:   "list-entity",
```

Comment:
just list?

Reply:
Defaulting to portfolio as list. Anything else, is list-type